opensrs/models.py

```diff
@@ -0,0 +1,12 @@
+from dateutil.parser import parse
+
+
+class Domain(object):
+    def __init__(self, data):
+        self.name = data['name']
+        self.auto_renew = data['f_auto_renew'] == 'Y'
+        self.expiry_date = parse(data['expiredate']).date()
+
+    @property
+    def tld(self):
+        return self.name.rsplit('.', 1)[1]
```

Comment:
I think this action is so cheap that it doesn't make sense to optimize it, and we can optimize number of symbols and improve readability.